src/Library/demos/User Info/main.blp

```diff
@@ -0,0 +1,70 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "User Info";
+  description: _("Request user information");
+  margin-top: 48;
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Label {
+      label: _("What best describes your request?");
+      margin-bottom: 18;
+      styles ["title-4"]
+    }
+
+    Box {
+      margin-bottom: 30;
+      halign: center;
+      spacing: 18;
+
+      Entry entry {
+        placeholder-text: _("Reason");
+      }
+
+      Button button {
+        label: _("Submit");
+        halign: center;
+        styles ["suggested-action"]
+      }
```

Comment:
can you use a AdwEntryRow and a "Request" button like ASHPD app?

![image](https://github.com/sonnyp/Workbench/assets/19673/bbf88cf3-3ffd-4a01-b3fa-d82d8c39cab1)

I would drop the question too